packages/venia-concept/src/components/ForgotPassword/FormSubmissionSuccessful/formSubmissionSuccessful.css

```diff
@@ -0,0 +1,13 @@
+.text {
+    padding: 0.5rem 1.5rem;
+    font-size: 0.875rem;
+    font-weight: 300;
+    line-height: 1.25rem;
+    text-align: center;
+    margin-bottom: 2rem;
+}
+
+.buttonContainer {
```

Comment:
Initially the idea was to create a component that can be used independently of the parent component. I believe it is better to specify this styling inside ForgotPasswordForm component.

Reply:
It is better to implement reusable components. From this perspective, it is better to leave `display: flex` inside ForgotPasswordForm and FormSubmissionSuccessful components.

Reply:
It just caught me by surprise that the two screens - which look so similar - were not using the same layout mechanism.

It works and looks great obviously but I was just trying to get out ahead of consistency issues, which just add time to future development tasks - I'm thinking about the next developer who has to modify this flow / these two components. It would be nice for this developer to work in a single layout strategy.

Honestly it's not a huge deal, just something that caught me by surprise so I thought I'd let you know about it 👍